articles/openshift/oshift-ref-lifecycle.md

```diff
@@ -29,22 +29,19 @@ Administrators and operators of the UKCloud for Managed OpenShift service.
 
 Red Hat's official versioning for their OpenShift Container Platform is _**X.Y**_ where _**X**_ indicates the major version and _**Y**_ indicates a minor version.
 
-Although Red Hat officially fully support each major _**X**_ version of the OpenShift Container Platform for five years, as a service provider, this duration is too long for UKCloud to ensure the highest levels of security, performance and features. As such, UKCloud will officially support the current **_(X.Y)_** and previous two minor versions **_(X.Y-1) and (X.Y-2)_**. As minor versions are released roughly every six months, this provides the assurance that your environments will be fully supported by UKCloud for a minimum of twelve months.
+Although Red Hat officially fully supports each major _**X**_ version of the OpenShift Container Platform for five years, as a service provider, this duration is too long for UKCloud to ensure the highest levels of security, performance and features. As such, UKCloud will officially support the current **_(X.Y)_** and previous minor version **_(X.Y-1)_**. As minor versions are released roughly every six months, this provides the assurance that your environments will be fully supported by UKCloud for a minimum of twelve months.
 
 ![Lifecycle](images/oshift-supported-versions.png)
 
 ### Upgrades of major _**X**_ and minor _**Y**_ versions
 
-Currently, UKCloud does not offer in-place upgrades. This provides you with total flexibility to upgrade at a time convenient to you, as well as providing the ability to test existing code and applications against any new versions prior to moving in to production.
+Currently, UKCloud does not offer in-place upgrades between major versions. Customers can trigger minor version upgrades inside the cluster at any time, providing total flexibility to upgrade at a convenient time.
 
-To facilitate any upgrades, UKCloud will deploy a new environment to customers based on the latest tested and approved release of OpenShift free of charge for thirty days in order to conduct any testing and migrate workloads over to the new environment. At the end of thirty days, UKCloud will shut down and remove the old environment to prevent any double-charging.
-
-> [!NOTE]
-> UKCloud will actively work with you during this period to help ensure a smooth migration and will not turn off any old environments until you advise us that it's safe to do so.
+We require customers to stay on UKCloud-supported versions, so ask that upgrades are done in a timely manner to stay on a Y-1 release. For more information on performing updates and what happens if customers aren't on Y-1, see [*Supported customer update strategy for version 4 clusters*](oshift-how-v4-update.md). <need update branch merged first to add URL to these docs>
```

Comment:
Presumably "<need update branch merged first to add URL to these docs>" should be removed now before we merge this branch?

Reply:
Thanks for spotting that. Annoyed with myself that I left it in :(